### PR TITLE
About us page

### PR DESCRIPTION
**Pull request by alma-castant**
About us page. Responding to: https://github.com/systers/portal/issues/77

### CODE REVIEW

File: systers_portal/common/tests/test_views.py [hunk 1]
```diff
@@ -12,3 +12,10 @@ def test_index_page(self):
         response = self.client.get(index_url)
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'common/index.html')
+
+    def test_about_us_page(self):
+        """Test about us page"""
+        index_url = reverse('about-us')
```
Comment (ana-balica) [inline systers_portal/common/tests/test_views.py]: I think you meant "about_us_url` :)

File: systers_portal/templates/common/about_us.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>About Systers</h1>
```
Comment (ana-balica) [inline systers_portal/templates/common/about_us.html]: May I suggest a `<hr>` after the heading?

File: systers_portal/templates/common/about_us.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+           alt="Systers Communities">
+    <div class="mt40"></div>
+    <p class="lead">
+      It’s important to know that you are not alone. Systers is a forum for all women involved in the technical aspects of computing. 
+      The list has over 4,000 members from at least 54 countries around the world. We welcome the participation of women technologists of
+      all ages and at any stage of their studies or careers. Systers is the world’s largest email community of women in technical roles in
+      computing. It was founded by Anita Borg in 1987 together with 12 other women as a small electronic mailing list for women in
+      “systems”.
+    </p>
+    <div class="mt40"></div>
```
Comment (ana-balica) [inline systers_portal/templates/common/about_us.html]: It's not necessary to add here space. The paragraphs have some margins to keep whitespace between them.

File: systers_portal/templates/common/about_us.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <p class="lead">
+      Systers Portal is a unified platform for Systers and its sub-groups to share information and get the latest news. Special Interest
+      communities are also offered where Systers can identify themselves culturally to share and offer support in challenges within
+      their communities.
+    </p>
+  </div>
+  <div class="mt40"></div>
```
Comment (ana-balica) [inline systers_portal/templates/common/about_us.html]: This block isn't adding any space between the footer and the container in full width. But it works when it's inside the container.

File: systers_portal/templates/common/about_us.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>About Systers</h1>
+    <img class="img-responsive" src="{% static 'img/about_systers.jpg' %}"
+           alt="Systers Communities">
+    <div class="mt40"></div>
+    <p class="lead">
```
Comment (ana-balica) [inline systers_portal/templates/common/about_us.html]: I am not sure if this `lead` class is perfect for long texts. How about trying to make it something similar to how blog entries look? We usually prefer to have Sans Serif titles and Serif fonts for body. You can create a new class that looks similar to `.blog-entry .body`.

Other than that, good job! Everything else is very good and thanks for remembering about tests.

**Comment by alma-castant** [comment]
@ana-balica I made the changes :) . I also cropped the image so it doesn't takes so much space

**Comment by ana-balica** [comment]
Accepted. Great job :+1:

**Pull request closed without merging**